reinforcement_learning/unit_test/live_model_test.cc

```diff
@@ -74,17 +74,14 @@ BOOST_AUTO_TEST_CASE(live_model_ranking_request)
 	ds.choose_rank(uuid, JSON_CONTEXT, response, &status);
 	BOOST_CHECK_EQUAL(status.get_error_code(), 0);
```

Comment:
This will send off the ranking request to the logger, I think we should put a wait in before tearing down the webserver

Reply:
In fact that may be the issue, that the log request is not able to make it to the web server because it is torn down almost instantly.

Reply:
Since the async_batcher destructor flushes to the network this change should fix it 👍

Reply:
Is a wait required here?  I'd like to avoid one if possible as they typically are really covering up for a wait condition.